src/apiCalls.js

```diff
@@ -18,18 +18,30 @@ export const fetchUsers = () => {
         }})
 }
 
-export const postFavorite = (user, movie) => {
-  const post = {
-      method: 'POST',
+export const addFavorite = (user, movie) => {
+  const patch = {
+      method: 'PATCH',
       headers: {'Content-Type': 'application/json'},
       body: JSON.stringify(movie)
     }
-  return fetch(`https://rancid-tomatillos-api-kd-lm.herokuapp.com/api/v1/users/${user.id}`, post)
+  return fetch(`https://rancid-tomatillos-api-kd-lm.herokuapp.com/api/v1/users/${user.id}`, patch)
     .then(res => {if (!res.ok) {
         return `${res.status} error. Sorry! Something went wrong! Try again later or go to Contact Us to contact the developers with questions!`;
       } else {
-        console.log('post response', res)
         return res.json();
       }})
+}
 
+export const deleteFavorite = (user, movie) => {
+  const deleteRequest = {
+      method: 'DELETE',
+      headers: {'Content-Type': 'application/json'},
+      body: JSON.stringify(movie)
+    }
+  return fetch(`https://rancid-tomatillos-api-kd-lm.herokuapp.com/api/v1/users/${user.id}`, deleteRequest)
+    .then(res => {if (!res.ok) {
+        return `${res.status} error. Sorry! Something went wrong! Try again later or go to Contact Us to contact the developers with questions!`;
+      } else {
+        return res.json();
+      }})
```

Comment:
Definitely think the delete is necessary for the flow of things! This is great! I think we could have done this with a PATCH(?) but getting this to work this way is awesome! Getting that practice with different methods is always great! Love that about coding, so many ways to do so many things!